omnigibson/envs/env_base.py

```diff
@@ -502,67 +581,52 @@ def step(self, action):
                 of actions
 
         Returns:
-            4-tuple:
+            5-tuple:
                 - dict: state, i.e. next observation
                 - float: reward, i.e. reward at this current timestep
-                - bool: done, i.e. whether this episode is terminated
+                - bool: terminated, i.e. whether this episode ended due to a failure or success
+                - bool: truncated, i.e. whether this episode ended due to a time limit etc.
```

Comment:
Doesn't this break gym-compatibility? Can we map these breakdowns into the `info` dict?

Reply:
This is the new gym / gymnasium API. There is a backwards compatibility layer provided in gymnasium that can be used to wrap this thing if need be.